roles/upload-site/tasks/main.yml

```diff
@@ -61,6 +61,19 @@
     - "-y updb"
   when: ignore_updb_errors == "TRUE"
 
+- name: Get current install profile
+  shell: "{{ drush_alias }} ev 'echo variable_get(\"install_profile\", \"set_me\");'"
```

Comment:
`drush ev` rather than `drush vget` here, so that you can set a default? SMRT.

Reply:
Yeah, I was disappointed that vget did not have a default option.